src/TestStack.White/InputDevices/BareMetalMouse.cs

```diff
@@ -0,0 +1,157 @@
+using System;
+using System.Runtime.InteropServices;
+using TestStack.White.WindowsAPI;
+
+namespace TestStack.White.InputDevices
+{
+    public class BareMetalMouse
+    {
+        #region DLL Import
+
+        [DllImport("user32", EntryPoint = "SendInput")]
+        protected static extern int SendInput(uint numberOfInputs, ref Input input, int structSize);
+
+        [DllImport("user32", EntryPoint = "SendInput")]
+        protected static extern int SendInput64(int numberOfInputs, ref Input64 input, int structSize);
+
+        [DllImport("user32.dll")]
+        protected static extern IntPtr GetMessageExtraInfo();
+
+        [DllImport("user32.dll")]
+        protected static extern bool GetCursorPos(ref System.Drawing.Point cursorInfo);
+
+        [DllImport("user32.dll")]
+        protected static extern bool SetCursorPos(int x, int y);
+
+        [DllImport("user32.dll")]
+        protected static extern bool GetCursorInfo(ref CursorInfo cursorInfo);
+
+        [DllImport("user32.dll")]
+        protected static extern short GetDoubleClickTime();
+
+        [DllImport("user32.dll")]
+        protected static extern int GetSystemMetrics(SystemMetric smIndex);
+
+        #endregion
+
+        #region Public
+
+        public static void LeftUp()
+        {
+            MouseButtonUp(MouseButton.Left);
+        }
+
+        public static void LeftDown()
+        {
+            MouseButtonDown(MouseButton.Left);
+        }
+
+        public static void RightUp()
```

Comment:
I think those static methods should be removed or at least be marked as obsolete.